src/TimeService.Data.Provider.MsSql.Ef/Migrations/20210518000000_InitialCreate.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using LT.DigitalOffice.TimeService.Models.Db;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.TimeService.Data.Provider.MsSql.Ef.Database.Migrations
+{
+    [DbContext(typeof(TimeServiceDbContext))]
+    [Migration("20210518000000_InitialCreate")]
+    public class InitialCreate : Migration
+    {
+        private const string ColumnIdName = "Id";
+
+        #region Create tables
+
+        private void CreateTableWorkTimes(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: DbWorkTime.TableName,
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false),
+                    ProjectId = table.Column<Guid>(nullable: false),
+                    CreatedBy = table.Column<Guid>(nullable: false),
+                    StartTime = table.Column<DateTime>(nullable: false),
+                    EndTime = table.Column<DateTime>(nullable: false),
+                    CreatedAt = table.Column<DateTime>(nullable: false),
+                    Title = table.Column<string>(nullable: false),
+                    Description = table.Column<string>(nullable: true),
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_WorkTimes", x => x.Id);
```

Comment:
you can call: $"PK_{DbWorkTime.TableName}", x => x.Id

Reply:
it is a very specific form of writing of keys, maybe you are not right @borovHokage